examples/ConfigurationStoreBackup/KeyLabel.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+
+namespace ConfigurationStoreBackup
+{
+    internal struct KeyLabel
+    {
+        public string Key { get; }
+        public string Label { get; }
+
+        public KeyLabel(string key, string label)
+        {
+            Key = key;
+            Label = label;
+        }
+
+        public override bool Equals(Object obj)
+        {
+            if (obj is KeyLabel)
+            {
+                KeyLabel keyLabel = (KeyLabel)obj;
```

Comment:
C# 7.0 and later support cast using pattern matching. So you could simplify this as `if (obj is KeyLabel keyLabel)`